Resources/Prototypes/Catalog/Fills/Lockers/medical.yml

```diff
@@ -2,6 +2,20 @@
   id: LockerMedicineFilled
   suffix: Filled
   parent: LockerMedicine
+  description: "Ze healing leaves little time for ze hurting."
+  components:
+  - type: StorageFill
+    contents:
+      - id: BoxPillCanister
```

Comment:
Why pill canisters? Doctors have nothing to fill them with and chemists have them in the chemistry closet.

Reply:
Medicine lockers are bit of buffet of medical junk, so I think it's reasonable to give the doctors some pill canisters so they can build their own pill collection.